src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -24,7 +24,13 @@ class ConfirmationBox extends React.Component {
 					this.props.name
 				}?`}</div>
 				<div className="confirmation-hand">
-					<Hand />
+					<Hand
+						snack={
+							this.props.storeList.find(
+								item => item.name === this.props.name
```

Comment:
Would be good to extract this, we could easily memoise it if we discovered it was a performance concern

```
const image = this.getImage(storeList, name)

snack={image}
```